function/auth.js

```diff
@@ -37,10 +42,13 @@ const authorize = async (appConfig = {}, authToken) => {
 const configuredAuth = (options = {}) => {
   return async (req, res, next) => {
     try {
-      const authToken = requireAuthorizationHeader(req);
-      await authorize(options, authToken);
+      requireAuthorizationHeader(req);
+      await authorize(options, req);
       next();
     } catch (err) {
+      const userEmail = req.email || 'unknown/anonymous user';
+      logger.error(`error during authorization: ${userEmail} requested ${req.originalUrl} ` +
+          `and resulted in ${JSON.stringify(err)}`);
```

Comment:
this does double-log in some cases. I'm okay with that ... I want this here to catch anything that got missed, but it's a generic message. In the double-logging cases the first message will be specific (e.g. `Rejecting request to ${req.originalUrl} from unwhitelisted user ${email}.`) and then followed by this generic one.